test/parallel/test-require-resolve.js

```diff
@@ -25,15 +25,15 @@ const fixtures = require('../common/fixtures');
 const assert = require('assert');
 
 assert.strictEqual(
-  fixtures.path('a.js').toLowerCase(),
-  require.resolve(fixtures.path('a')).toLowerCase());
+  require.resolve(fixtures.path('a')).toLowerCase(),
+  fixtures.path('a.js').toLowerCase());
 assert.strictEqual(
-  fixtures.path('a.js').toLowerCase(),
-  require.resolve(fixtures.path('a')).toLowerCase());
+  require.resolve(fixtures.path('a')).toLowerCase(),
+  fixtures.path('a.js').toLowerCase());
```

Comment:
looks like the above 2 asserts are identical. Not an issue of this PR, but does any one know whether it is intentional vs. accidental?

Reply:
good catch @gireeshpunathil, that change was introduced in https://github.com/nodejs/node/commit/91a465c460a1525103611a03ca277b061f320228#diff-a00e02f229f04b6262da068e8df33ed5 and the version before that is https://github.com/nodejs/node/commit/b1506f713c8866c47d17b0d7fe76fd5c6697ef42#diff-a00e02f229f04b6262da068e8df33ed5

Reply:
updated and rebased 👍